CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changes
 
+## 0.5.0
```

Comment:
We typically [like `UNRELEASED`](https://github.com/FormidableLabs/spectacle-cli/blob/master/CONTRIBUTING.md#before-submitting-a-pr) rather than a specific version, then following the release process in https://github.com/FormidableLabs/spectacle-cli/blob/master/CONTRIBUTING.md#releasing-a-new-version-to-npm